openstack/imageservice/v2/images/requests.go

```diff
@@ -37,7 +48,9 @@ type ListOpts struct {
 	Owner string `q:"owner"`
 
 	// Status filters on the status of the image.
-	Status ImageStatus `q:"status"`
+	// Multiple statuses can be specified by constructing a string
+	// such as "in:saving,queued".
+	Status string `q:"status"`
```

Comment:
It seems like users could still create their own `ImageStatus` by wrapping a string, right? Or would this not work:
```go
imageStatus := images.ImageStatus("in:saving,queued")
```
?

Reply:
Ah, good call. I'll change this.